rules-emerging-threats/2023/Exploits/CVE-2023-23397/win_smbclient_connectivity_exploit_cve_2023_23397_outlook_remote_file.yml

```
@@ -25,18 +26,53 @@ detection:
             # - 31001 # Error (Doesn't contain the "ServerAddress" field)
     filter_main_local_ips:
         - ServerAddress|cidr:
+              # IPv4
               - '10.0.0.0/8'
               - '127.0.0.0/8'
               - '169.254.0.0/16'
               - '172.16.0.0/12'
               - '192.168.0.0/16'
+              # IPv6
+              - '::1/128'  # IPv6 loopback
+              - 'fe80::/10'  # IPv6 link-local addresses
+              - 'fc00::/7'  # IPv6 private addresses
         - Address|startswith:
         # This is for EventID 30804, which doesn't have the "ServerAddress" field, but a field called "Address" and it contains a socket address (IP:Port) and not an IP
+              # IPv4
               - '10.'
               - '127.'
               - '169.254.'
               - '172.'
               - '192.168.'
+              # IPv6
+              - '::1'
+              - 'fe80::'
+              - 'fc00::'
+        # The filters below cover the XML raw log
+        - Address|contains:
+              # IPv6
+              - '00000000000000000000000000000001' # ::1 - IPv6 loopback
+              - 'FE80000000000000' # fe80:: - IPv6 link-local addresses
+              - 'FC00000000000000' # fc00:: - IPv6 private addresses
+              # IPv4
+              # The "?" are meant to represent the port
+              - '0200????C0A8' # 192.168.
+              - '0200????AC' # 172.
+              - '0200????0A' # 10.
+              - '0200????7F' # 127
+              - '0200????A9FE' # 169.254.
+        - ServerAddress|contains:
+              # IPv6
+              - '00000000000000000000000000000001' # ::1 - IPv6 loopback
+              - 'FE80000000000000' # fe80:: - IPv6 link-local addresses
+              - 'FC00000000000000' # fc00:: - IPv6 private addresses
+              # IPv4
+              # The "?" are meant to represent the port
+              - '0200????C0A8' # 192.168.
+              - '0200????AC' # 172.
+              - '0200????0A' # 10.
+              - '0200????7F' # 127
```

Should not be like this
```suggestion
              - '0200??????????AC' # 172.
              - '0200??????????0A' # 10.
              - '0200??????????7F' # 127
```
"????" for Port
"??????" for 3 last bytes of the IP V4
as the `?` is for 1 character